calibration/CTDGVM/CGINS-CTDGVM-09059__20180109.csv

```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9059,CC_calibration_date,1515456000000,sensor calibration date (milliseconds since 1970-01-01)
```

Comment:
looks good - are we putting file_source info in (same question for all)?

Reply:
Looks ok, but I have the same question.